src/Core/components/MathList.js

```diff
@@ -55,29 +52,13 @@ export default class MathList extends InlineComponent {
     let sugarInstructions = super.returnSugarInstructions();
 
     let groupIntoMathsSeparatedBySpaces =
-      returnGroupIntoComponentTypeSeparatedBySpaces({
-        componentType: "math",
-      });
-    let breakStringsIntoMathsBySpaces =
-      returnBreakStringsIntoComponentTypeBySpaces({
+      returnGroupIntoComponentTypeSeparatedBySpacesOutsideParens({
```

Comment:
holy descriptive name batman

Reply:
Hah!  Yeah, I kept getting confused by different break strings functions, so this is my effort to keep them apart.